packages/events-example/src/commands/reconcile/index.ts

```diff
@@ -0,0 +1,209 @@
+import { Command, Flags } from '@oclif/core'
+import Nile, { Instance, NileApi } from '@theniledev/js';
+
+import * as aws from '@pulumi/aws';
+import { PolicyDocument } from "@pulumi/aws/iam";
+import { DestroyResult, InlineProgramArgs, LocalWorkspace, StackSummary, Stack, UpResult, PulumiFn } from "@pulumi/pulumi/automation";
+
+export default class Reconcile extends Command {
+  static description = 'reconcile nile/pulumi deploys'
+
+  static flags = {
+    basePath: Flags.string({ char: 'b', description: 'basePath', default: 'http://localhost:8080' }),
+    workspace: Flags.string({ char: 'w', description: 'workspace', default: 'tryhard' }),
+    email: Flags.string({ char: 'e', description: 'email', default: 'trying@demo.com' }),
+    password: Flags.string({ char: 'p', description: 'password', default: 'trying' }),
+    status: Flags.boolean({ char: 's', description: 'status', default: false })
+  }
+
+  static args = [{ name: 'person', description: 'Person to say hello to', required: false }]
+
+  localWorkspace!: LocalWorkspace;
+  nile!: NileApi;
+
+  async waitOnStack(stack: Stack): Promise<void> {
+    var stackInfo;
+    do {
+      stackInfo = await stack.info();
+      console.log(stackInfo);
+    } while (stackInfo != undefined && stackInfo?.result !== "succeeded");
+  }
+
+  async getStack(stackName: string, program: PulumiFn): Promise<Stack> {
+    const args: InlineProgramArgs = {
+      stackName,
+      projectName: "tryhard",
```

Comment:
its up to you if you want to support the flags/args here. At the very least, I would consolidate all of the strings we're using so that it is possible to check this out, change it to the strings I want, and get it working.